TINY-8122: Word Count is treating characters either side of a word as their own words (#8512)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

> - Resolved a bug that caused inaccurate word counts for documents with specific characters. #TINY-8122

Reviewer: This should go under `fixed`

- Inserting a link for a selection from quickbars didn't preserve formatting. #TINY-9593
- Inline dialog position was not correct when the editor wasn't inline and was contained in a `fixed` or `absolute` positioned element. #TINY-9554
- Sticky toolbars would not have fade transition when undocking in classic iframe mode. #TINY-9408

> - Resolved a bug that caused inaccurate word counts for documents with specific characters. #TINY-8122

Reviewer 2: We normally bug fix log messages like that. Compare to the other ones. Maybe this is better not sure:
```suggestion
- Word count was inaccurate for documents with specific characters. #TINY-8122
```
It is a bit generic but listing the characters seems tricky as well since they are so weird.

Reviewer: yes we're in the "fixed" section of the changes, we don't need to include the phrase "resolved a bug" in each entry

- Opening color swatches would cause tab to crash when `color_cols` or other column option was set to 0. #TINY-9649
- Closing a view, `More...` button disappeared if the editor had `toolbar_mode: 'sliding'` and the toolbar was opened. #TINY-9419
- Inline dialogs would open partially off screen when the toolbar had a small width. #TINY-9588
- Word count was inaccurate for documents with specific characters. #TINY-8122

> 

Reviewer 3: ```suggestion
```
